extensions-contrib/dropwizard-emitter/src/main/java/org/apache/druid/emitter/dropwizard/DropwizardEmitterConfig.java

```diff
@@ -0,0 +1,136 @@
+ * under the License.
+ */
+
+package org.apache.druid.emitter.dropwizard;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+
+public class DropwizardEmitterConfig
+{
+  private static int DEFAULT_MAX_GAUGE_COUNT = 100_000;
+  @JsonProperty
+  private final List<DropwizardReporter> reporters;
+  @JsonProperty
+  private final String prefix;
+  @JsonProperty
+  private final Boolean includeHost;
+  @JsonProperty
+  private final String dimensionMapPath;
+  @JsonProperty
+  private final List<String> alertEmitters;
+  @JsonProperty
+  private final int maxGaugeCount;
+
+  @JsonCreator
+  public DropwizardEmitterConfig(
+      @JsonProperty("reporters") List<DropwizardReporter> reporters,
+      @JsonProperty("prefix") String prefix,
+      @JsonProperty("includeHost") Boolean includeHost,
+      @JsonProperty("dimensionMapPath") String dimensionMapPath,
+      @JsonProperty("alertEmitters") List<String> alertEmitters,
+      @JsonProperty("maxGaugeCount") Integer maxGaugeCount
+  )
+  {
+    this.reporters = reporters;
+    this.prefix = prefix;
+    this.alertEmitters = alertEmitters == null ? Collections.emptyList() : alertEmitters;
+    this.includeHost = includeHost != null ? includeHost : true;
+    this.dimensionMapPath = dimensionMapPath;
+    this.maxGaugeCount = maxGaugeCount == null ? DEFAULT_MAX_GAUGE_COUNT : maxGaugeCount;
+  }
+
+  @JsonProperty
+  public List<DropwizardReporter> getReporters()
+  {
+    return reporters;
+  }
+
+  @JsonProperty
+  public String getPrefix()
+  {
+    return prefix;
+  }
+
+  @JsonProperty
+  public Boolean getIncludeHost()
+  {
+    return includeHost;
+  }
+
+  @JsonProperty
+  public String getDimensionMapPath()
+  {
+    return dimensionMapPath;
+  }
+
+  @JsonProperty
+  public List<String> getAlertEmitters()
+  {
+    return alertEmitters;
+  }
+
+  @JsonProperty
+  public int getMaxGaugeCount()
+  {
+    return maxGaugeCount;
+  }
+
+  @Override
+  public boolean equals(Object o)
+  {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    DropwizardEmitterConfig that = (DropwizardEmitterConfig) o;
+    return maxGaugeCount == that.maxGaugeCount &&
+           Objects.equals(reporters, that.reporters) &&
+           Objects.equals(prefix, that.prefix) &&
+           Objects.equals(includeHost, that.includeHost) &&
+           Objects.equals(dimensionMapPath, that.dimensionMapPath) &&
+           Objects.equals(alertEmitters, that.alertEmitters);
+  }
+
+  @Override
+  public int hashCode()
+  {
+    return Objects.hash(reporters, prefix, includeHost, dimensionMapPath, alertEmitters, maxGaugeCount);
+  }
+
+  @Override
+  public String toString()
+  {
+    return "DropwizardEmitterConfig{" +
+           "reporters=" + reporters +
```

Comment:
`toString()` is not implemented for `DropwizardReporter`s.

Reply:
done.